server.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require('supertest');
+const server = require('./server');
+const db = require('./dbConfig');
+
+
+
+afterEach(async() => {
+    await db('games').truncate();
+})
+
+describe('Route handlers', () =>{
+    
+    describe('Post endpoint for games', () =>{
+        it('Should respond with a 422 status code if information is missing', async() =>{
```

Comment:
The test for 422 status code can be split up into two tests: one to test the missing genre in request body and the other to test the missing title in the request body